EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,167 @@
+    <tr>
+        <td><strong>Normative Keywords &amp; Statements</strong></td>
+        <td>
+            MUST, MAY, MUST NOT and normative statements
+        </td>
+        <td>
+            This column:</br>
+            &nbsp;&nbsp; * MUST NOT be null when ChargeClass is ... </br>
+            &nbsp;&nbsp; * MUST be null when ChargeCategory is ... </br>
+            &nbsp;&nbsp; * MAY be null for all other combinations of ... </br>
+        </td>
+        <td>
+           - All uppercase, without bold.<br>
+           - Bullet list format. <br>
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Glossary</strong></td>
+        <td>
+            [SKU](), [resource](), [service]()
+        </td>
+        <td>
+            [*SKU*](#glossary:sku) <br>
+            [*resource*](#glossary:resource) <br>
+            [*service*](#glossary:service) <br>
+        </td>
+        <td>
+            - Blue font + italic<br>
+            - The first ocurrence in a section is linked to the glossary.
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Important Text</strong></td>
+        <td> <img width="492" alt="image" src="https://github.com/user-attachments/assets/c6f60ff9-1503-43a3-8229-004595b334d2"></td>
+        <td>> Important Consideration</td>
+        <td>- It is added as a note.</td>
+    </tr>
+    <tr>
+        <td><strong>Key-Value Format</strong></td>
+        <td>JSON Script</td>
+        <td>
+            <pre>
+              **Example**:
+              ```json
+                {
+                    "key1": "value1",
+                    "key2": true,
+                    "key3": 123
+                }
+              ```
+            </pre>
+        </td>
+        <td>- Monospace font</td>
+    </tr>
+    <tr>
+      <td><strong>Tables</strong></td>
+      <td><img width="492" alt="image" src="https://github.com/user-attachments/assets/5185cbf9-306d-4663-a1c7-c8b7ab5c5bb8"></td>
+      <td><img width="492" alt="image" src="https://github.com/user-attachments/assets/83d0977f-a731-4def-93e3-b3e5f5dedb72"></td>
+      <td> - Tables</td>
+    </tr>
+</table>
+
+**Editorial Notes**
+* **Linking Only the First Time**: To prevent excessive linking within sections, Column Name, Column ID, Attributes Names, Attributes IDs, and Glossary will only be linked to their corresponding section or glossary the first time they appear in a section.
+
+* **Normative Requirements as a Bullet List**: Normative statements (those using Normative Keywords) should be written as bullet points instead of lengthy sentences. 
+
+### Example
+
+>**2.28. Pricing Quantity**
+>
+>The **[Pricing Quantity](#pricing-quantity)** represents the volume of a given [SKU]() associated with a [resource]() or [service]() used or purchased, based on the **[Pricing Unit](#pricing-unit)**. Distinct from **[Consumed Quantity](#consumed-quantity)** (complementary to **[Consumed Unit](#consumed-unit)**), it focuses on pricing and cost, not resource and service consumption. 
```

Comment:
This is missing the link for SKU. Not sure if that's in the original file this was copied from.